content/en/blog/_posts/2024-04-03-windows-ops-readiness.md

```diff
@@ -0,0 +1,158 @@
+| `Extend.ActiveDirectory` | Tests features related to Active Directory functionality.                                                                           |
+| `Extend.NetworkPolicy`   | Tests features related to Network Policy functionality.                                                                             |
+| `Extend.Network`         | Tests advanced networking functionality, (ability to support IPv6)                                                                  |
+| `Extend.Worker`          | Tests features related to Windows worker node functionality, (ability for nodes to access TCP and UDP services in the same cluster) |
+
+## How to conduct operational readiness tests for Windows nodes
+
+To run the Windows Operational Readiness test suite, refer to the test suite's
+[`README`](https://github.com/kubernetes-sigs/windows-operational-readiness/blob/main/README.md), which explains how to set it up and run it. The test suite offers 
+flexibility in how you can execute tests, either using a compiled binary or a 
+Sonobuoy plugin. You also have the choice to run the tests against the entire 
+test suite or by specifying a list of categories. Cloud providers have the 
+choice of uploading their conformance results, enhancing transparency and reliability.
+
+Once you have checked out that code, you can run a test. For example, this sample 
+command runs the tests from the `Core.Concurrent` category:
+
+```shell
+$ ./op-readiness --kubeconfig $KUBE_CONFIG --category Core.Concurrent
+```
+
+If you want to test your changes against a specific pull request using the 
+Windows Operational Readiness Specification, use the following bot command in 
+the new pull request.
+
+```
+/test operational-tests-capz-windows-2019
+```
+
+## Looking ahead
+
+We’re looking to improve our curated list of Windows-specific tests by adding 
+new tests to the Kubernetes repository and also identifying existing test cases 
+that can be targetted. The long term goal for this subproject is to continually 
+enhance test coverage for Windows worker nodes and improve the robustness of 
+Windows support, facilitating a seamless experience across diverse cloud 
+environments. We also have plans to integrate the Windows Operational Readiness 
+tests into the official Kubernetes conformance suite.
+
+If you are interested in helping us out, please reach out to us! We welcome help 
+in any form, from giving once-off feedback to making a code contribution, 
+to having long-term owners to help us drive changes. The Windows Operational 
+Readiness is owned by the SIG Windows team as a sub-project. You can reach out 
+to the team on the [Kubernetes Slack workspace](https://slack.k8s.io/) **#sig-windows** 
+channel. You can also explore the [Windows Operational Readiness test suite](https://raw.githubusercontent.com/kubernetes-sigs/windows-operational-readiness) 
```

Comment:
This link doesn't seem right. How about https://github.com/kubernetes-sigs/windows-operational-readiness/#readme?